src/main/java/com/stripe/model/reporting/ReportRun.java

```diff
@@ -0,0 +1,103 @@
+package com.stripe.model.reporting;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.ExpandableField;
+import com.stripe.model.FileUpload;
+import com.stripe.model.HasId;
+import com.stripe.model.MetadataStore;
+import com.stripe.model.StripeObject;
+import com.stripe.net.ApiResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class ReportRun extends ApiResource implements HasId {
+  @Getter(onMethod = @__({@Override})) String id;
+  String object;
+  Long created;
+  String error;
+  Boolean livemode;
```

Comment:
I think that these actually don't have a livemode property :/

Reply:
The `ReportRun` resource does. It was added a few hours ago as I asked for it

Reply:
Ah you're right. I see it in there now.